docs/feature-gates.md

```diff
@@ -96,6 +97,19 @@ which is closer to its origin with this feature. Refer to this
 [link](https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/)
 for more information.
 
+
+### ProxyTerminatingEndpoints
+
+`ProxyTerminatingEndpoints` enables ProxyTerminatingEndpoints support in AntreaProxy. 
+The feature ProxyTerminatingEndpoints is at beta stage in Kubernetes 1.22 (beta), and
+it is enabled by default in Kubernetes 1.26. Terminating Endpoints are now included in
+EndpointSlice, with the addition of two new Endpoint conditions "Serving" and "Terminating".
+Even though the EndpointSlice API now includes terminating Endpoints, AntreaProxy only
```

Comment:
This sentence is a little misleading. Consider: "When `ProxyTerminatingEndpoints` is not enabled, AntreaProxy only forwards Service traffic to "Ready" Pods that are not terminating; when the feature is enabled, AntreaProxy can forward traffic to terminating Endpoints if there is no ready Endpoint".

Reply:
Done